plugin/lspconfig.lua

```diff
@@ -3,6 +3,10 @@ if vim.g.lspconfig ~= nil then
 end
 vim.g.lspconfig = 1
 
+if vim.fn.has('nvim-0.12') == 1 then
```

Comment:
could check `2 == vim.fn.exists(':lsp')` and then can merge this without waiting :)

Reply:
see `:cmdname` note in `:h exists()`. probably should clarify the docs

Reply:
I see. Just scrolled down to the return value and didn't read the whole thing.